### PR TITLE
Implement the 'Region Line Color' setting

### PR DESCRIPTION
**Pull request by MikeIsAStar**

### CODE REVIEW

File: payload/game/ui/ctrl/CtrlRaceNameBalloon.cc [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 #include "game/system/RaceConfig.hh"
 #include "game/system/SaveManager.hh"
+#include "game/ui/page/RacePage.hh"
 #include "game/ui/SectionManager.hh"
 #include "game/ui/TeamColors.hh"
-#include "game/ui/page/RacePage.hh"
```
Comment (stblr) [inline payload/game/ui/ctrl/CtrlRaceNameBalloon.cc:7]: This should be controlled by `SortIncludes` in clang-format.

File: payload/game/ui/ctrl/CtrlRaceNameBalloon.cc [hunk 2]
```diff
@@ -54,21 +54,19 @@ void CtrlRaceNameBalloon::refresh(u32 playerId) {
...
-            { 232, 212, 0, 255 },
-            { 9, 170, 255, 255 },
-            { 255, 39, 127, 255 },
-            { 38, 220, 65, 255 },
-        };
-        color = colors[playerId];
+        color = TeamColors::Get(
+                static_cast<u32>(
+                        saveManager->getSetting<SP::ClientSettings::Setting::RegionLineColor>()) ^
+                1);
```
Comment (stblr) [inline payload/game/ui/ctrl/CtrlRaceNameBalloon.cc:69]: Maybe make a temporary because that auto-formatting is ugly.

**Review by stblr — CHANGES_REQUESTED** [review]

File: payload/game/ui/ctrl/CtrlRaceNameBalloon.cc (hunk 1 above)
Comment (MikeIsAStar) [reply payload/game/ui/ctrl/CtrlRaceNameBalloon.cc]: Resolved in commit [8a01170](https://github.com/stblr/mkw-sp/pull/420/commits/8a01170d08cdcf172924fe29faa67a9d936bb200).

File: payload/game/ui/ctrl/CtrlRaceNameBalloon.cc (hunk 2 above)
Comment (MikeIsAStar) [reply payload/game/ui/ctrl/CtrlRaceNameBalloon.cc]: Resolved in commit [08a4e81](https://github.com/stblr/mkw-sp/pull/420/commits/08a4e81afbc9ea009a9be8a4048ce9ea3c9b8d6d).

**Review by stblr — APPROVED** [review]

**Pull request merged**